src/lib/db/clients/mysql.js

```diff
@@ -277,7 +277,7 @@ export function query(conn, queryText) {
 export async function updateValues(conn, updates) {
   const updateCommands = updates.map(update => {
     return {
-      query: `UPDATE \`${update.table}\` SET \`${update.column}\` = ? WHERE \`${update.pkColumn}\` = ?`,
+      query: `UPDATE ${wrapIdentifier(update.table)} SET ${wrapIdentifier(update.column)} = ? WHERE ${wrapIdentifier(update.pkColumn)} = ?`,
```

Comment:
I think we need the backticks if someone names a table with (reserved terms) `update`, `table`, etc.

Reply:
I've included the library `knex` in the project and want to use that for all the update queries

Reply:
I'll move mysql and sqlite to using that sql generator over the next few days. I just pushed sqlserver and pg support using it. Much simplifies the code. I hate manually building sql.

Oh, the `wrapIdentifier` adds the backticks fyi.

Reply:
Oh good to know :+1: